### DISCUSSION TITLE
Admin user status changed to InActive

### DISCUSSION THREAD

**Discussion by nCubed** [keygen-sh/community#48] [General]
I understand a user's status will be changed to `INACTIVE` after 90 days of creation or after 90 days of no license validations. 

I'm wondering if users in the admin role should be excluded from the policy to auto change their status from active to inactive?

For example, our app will validate the admin user exists in Keygen as well as validates their status; when their status is anything other than `ACTIVE`, then they are excluded from certain operations. 

What do you think?

**Comment by ezekg** [comment]
Some admin users are licensed users, and need to be treated as such, so this won't work. A user's `status` applies to any user, and simply signals if any of their licenses have been active within the last 90 days, regardless of role. A user's `status` has no effect on what the user can do. If you need this functionality, you can add a special case in your operation based on a user's role.
